### PR TITLE
Depends On: #27 Driver container base

### PR DESCRIPTION
**Pull request by zvonkok**
Hiding all the gory details from customers and building a DriverContainer base image. 
Added a new recipe to build the driver-container-base. 

```
 SPECIALRESOURCE=driver-container-base PULLPOLICY=Always make deploy
```

### CODE REVIEW

File: deploy/operator.yaml [hunk 1]
```diff
@@ -23,9 +23,6 @@ spec:
           imagePullPolicy: Always
           env:
             - name: WATCH_NAMESPACE
-              valueFrom:
-                fieldRef:
-                  fieldPath: metadata.namespace
```
Comment (kpouget) [inline deploy/operator.yaml:28]: no value for this `env` variable? I'm not sure that was intended...

**Comment by kpouget** [comment]
The PR defines a new `specialresource` (`driver-container-base`) and adds the manifests and operations to build it as a `ubi8` image containing the [kmods-via-containers](https://github.com/kmods-via-containers/kmods-via-containers) installation.

Apart from a minor inconsistency mentioned in the comment above,
/lgtm

File: recipes/driver-container-base/manifests/0000-driver-container-base-build.yaml [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+    WORKINGDIR() { buildah --storage-driver vfs config --workingdir $1 $CNT; }
+    PUSH() { buildah --storage-driver vfs push --tls-verify=false --authfile /root/.dockercfg  $@; }
+
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{.SpecialResource.Name}}-install-dependencies
+data:
+  install-depedencies.sh: |-
```
Comment (dagrayvid) [inline recipes/driver-container-base/manifests/0000-driver-container-base-build.yaml:40]: 's/depedencies/dependencies/'

This typo is also repeated in the next configmap and the pod .yaml in this file. Other than that lgtm.

File: deploy/operator.yaml (hunk 1 above)
Comment (zvonkok) [reply deploy/operator.yaml:28]: Yes it is. SRO will watch all namespaces for SpecialResource CRs. Will remove the comments.

Comment (zvonkok) [reply deploy/operator.yaml:28]: If it is empty -> All Namespaces, otherwise it uses the namespace from the metadata section.

Comment (kpouget) [reply deploy/operator.yaml:28]: ok, thanks for the explanation

**Pull request merged**